spec/spec_helper.rb

```diff
@@ -114,6 +114,8 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+  require 'webmock/rspec'
```

Comment:
Unless this `require` is conditional, I think it would be surprising to see it here. Wouldn't necessary think to look for it somewhere other than the top.

Reply:
I'll move it, but I think this is where it's in MCFE, but that was probably added in the wrong spot also by me